packages/dd-trace/src/config.js

```diff
@@ -937,6 +950,15 @@ class Config {
     this._setBoolean(opts, 'traceId128BitGenerationEnabled', options.traceId128BitGenerationEnabled)
     this._setBoolean(opts, 'traceId128BitLoggingEnabled', options.traceId128BitLoggingEnabled)
     this._setString(opts, 'version', options.version || tags.version)
+
+    // For LLMObs, we want the environment variable to take precedence over the options.
+    // This is reliant on environment config being set before options.
+    // This is to make sure the origins of each value are tracked appropriately for telemetry.
+    // We'll only set `llmobs.enabled` on the opts when it's not set on the environment, and options.llmobs is provided.
+    const llmobsEnabledEnv = this._env['llmobs.enabled']
+    if (llmobsEnabledEnv == null && options.llmobs) {
+      this._setBoolean(opts, 'llmobs.enabled', !!options.llmobs)
```

Comment:
```suggestion
      this._setBoolean(opts, 'llmobs.enabled', options.llmobs)
```
`[_setBoolean](https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/config.js#L1111)` does `!!`

Reply:
I tried without `!!`, but found the problem was it would skip the `if (value === undefined || value === null)` check, but then would also skip `isTrue` and `isFalse` because both of them evaluate the truthiness of `value` by stringifying it and then checking it against either `true` or `1` (for `isTrue`) or `false` or `0` (for `isFalse`). 

Since `options.llmobs` is an object, when it's stringified, it just becomes `[object Object]`, and it doesn't hit any of the checks 😢 i don't see in `_setBoolean` where it applies the `!!` automatically

Reply:
make sense! thanks Sam :)